packages/sdk/src/cli/checkUnsupportedDataProxy.ts

```diff
@@ -0,0 +1,76 @@
+import chalk from 'chalk'
+import { O } from 'ts-toolbelt'
+
+import { getConfig, getSchemaPath, link } from '..'
+import { loadEnvFile } from '../utils/loadEnvFile'
+
+/**
+ * These are the cli args that we check the data proxy for. If in use
+ */
+const checkedArgs = {
+  '--url': true,
+  '--to-url': true,
+  '--from-url': true,
+  '--shadow-database-url': true,
+  '--schema': true,
+  '--from-schema-datamodel': true,
+  '--to-schema-datamodel': true,
+}
+
+type Args = O.Optional<O.Update<typeof checkedArgs, any, string>>
+
+/**
+ * Get the message to display when a command is forbidden with a data proxy flag
+ * @param command the cli command (eg. db push)
+ * @returns
+ */
+export const forbiddenCmdWithDataProxyFlagMessage = (command: string) => `
+${chalk.green(`prisma ${command}`)} command is not yet available with ${chalk.green('prisma://')}.
+
+More information in our documentation:
+${link('https://pris.ly/d/data-proxy')}
```

Comment:
We should probably have an anchor/section on that page that explicitly documents the current workaround (I think we advise people to have a different env var and then override `DATABASE_URL` before running `migrate` etc.

Easiest way to prepare for that is to create a new pris.ly link that we can then change to that headline/anchor when it exists.

Suggestions for naming? `/d/data-proxy-???`?

Reply:
/d/data-proxy-limitations with a page with all the current limitations + workarounds?

Reply:
That could work. `data-proxy-cli` could be a less judgy alternative 😆

Reply:
`data-proxy-cli` will be.

Reply:
I will set up the redirect on pris.ly 👍

Reply:
Done: https://pris.ly/d/data-proxy-cli